# Dr. Geoffrey Pawlak | PharmD, PhD
### Oncology Bioinformatics Researcher

I focus on identifying actionable biological levers in oncology by leveraging **DNA-seq** and **RNA-seq** data at **bulk** or **single-cell** level. My approach prioritizes high-performance, interoperable, and reproducible computational frameworks to bridge the gap between dry-lab findings and clinical applications.

---

## Research Focus & Vision
* **Current Research:** Decoding complex oncogenic mechanisms through multi-omics integration and robust statistical modeling of bulk sequencing data.
* **Strategic Direction:** Transitioning towards **Single-Cell Technology Integration** to resolve intratumoral heterogeneity and refine precision oncology strategies.

---

## Computational Expertise

![R](https://img.shields.io/badge/R-276DC3?style=for-the-badge&logo=r&logoColor=white) 
![Python](https://img.shields.io/badge/Python-3776AB?style=for-the-badge&logo=python&logoColor=white) 
![Bash](https://img.shields.io/badge/Shell_Script-4EAA25?style=for-the-badge&logo=gnu-bash&logoColor=white)
![Snakemake](https://img.shields.io/badge/Snakemake-000000?style=for-the-badge&logo=python&logoColor=white)
![Docker](https://img.shields.io/badge/Docker-2496ED?style=for-the-badge&logo=docker&logoColor=white)
![HPC](https://img.shields.io/badge/HPC-555555?style=for-the-badge&logo=linux&logoColor=white)

* **R & Bioconductor:** Expert in high-performance R coding (S4/R6). I strictly follow `snake_case` conventions, ensuring full English documentation and code interoperability for the scientific community.
* **Python:** Focused on large-scale data manipulation and transitioning towards single-cell technology integration (Scanpy/AnnData).
* **Workflows & Automation:** Advanced implementation of **Snakemake** pipelines for reproducible DNA/RNA-seq analysis.
* **Infrastructure & Scalability:** Expert use of **Docker** for containerization and **High-performance computing (HPC)** for managing intensive bioinformatic workloads.
* **Bash:** Mastering CLI environments for efficient file system management and job scheduling in cluster environments.

---

## Scientific Contributions & Projects

### Socials
[![Google Scholar](https://img.shields.io/badge/Google_Scholar-4285F4?style=flat-square&logo=google-scholar&logoColor=white)](https://scholar.google.com/citations?user=cgitYGIAAAAJ&hl=fr&oi=ao)
[![ORCID](https://img.shields.io/badge/ORCID-A6CE39?style=flat-square&logo=orcid&logoColor=white)](https://orcid.org/0000-0001-8882-1441)


### Example of research projects

#### **[System biology approch on cancers]**

GBM-cRegMap: A coregulatory influence map of glioblastoma (GBM) 


<p align="center"> 
  <img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41698-025-00890-0/MediaObjects/41698_2025_890_Fig1_HTML.png?as=webp" width=40% height=40%>
</p>
<p align="center">
  <em>PharmD practical project outline.</em>
</p>

Objective: Developing a unified network influence map to navigate GBM heterogeneity and plasticity.

Methodology: Used representation learning and the h-LICORN algorithm to infer a highly specific coregulatory network (GBM-CoRegNet) from 1,612 tumors.

Key Findings: Refined GBM classification into seven molecular classes , identifying the CL-C "cilium-rich" subclass as an intermediate state linked to chemotherapy resistance.

Regulators: Identified PAX8 and NKX2.5 as master regulators; NKX2.5 negatively impacts survival by regulating glucose metabolism.

#### **[Therapeutic repurposing pipeline]**

<p align="center"> 
  <img src="https://github.com/pawlakG/pharmD/blob/f141e3fb89491a4eb6d30d92290f41df89e236b8/output/img/pharmD_Outline.png" width=40% height=40%>
</p>
<p align="center">
  <em>PharmD practical project outline.</em>
</p>

his project implements a virtual screening and machine learning workflow to identify high-affinity, low-toxicity inhibitors targeting the MET receptor.

Architecture: Integration of ChEMBL and PubChem databases with structural filtering based on Lipinskiâ€™s rules and undesirable motif exclusion.

Methodology: Deep Learning approach using a Neural Network (pIC50â€‹ prediction MAE: 0.68) and the eToxPred algorithm to prioritize potent candidates with favorable safety profiles.

Pragmatic Approach: Focuses on narrowing massive chemical libraries down to a validated subset of 1,344 compounds to minimize experimental timeframe and budget.

---

## ðŸ“« Connect with me
[![LinkedIn](https://img.shields.io/badge/LinkedIn-0077B5?style=flat-square&logo=linkedin&logoColor=white)]([VOTRE_LIEN_LINKEDIN](https://www.linkedin.com/in/geopaw/))
[![Email](https://img.shields.io/badge/Email-D14836?style=flat-square&logo=gmail&logoColor=white)](mailto:pawlak.geoffrey@gmail.com)